packages/terra-tabs/src/Tabs.scss

```diff
@@ -0,0 +1,105 @@
+@import './ModularTabs';
+@import './StructuralTabs';
+
+:local {
+  .tabs-container {
+    width: 100%;
+  }
+
+  .collapsible-tabs-container,
+  .collapsed-tabs-container {
+    display: flex;
+    flex: 1 1 auto;
+    flex-wrap: nowrap;
+    justify-content: flex-start;
+    position: relative;
+    width: 100%;
+
+    &:focus {
+      outline: none;
+    }
+  }
+
+  .collapsible-tabs-container.is-calculating {
+    visibility: hidden;
+
+    /*
+     * During calculations tabs should display with auto size and not grow or shrink in order to get
+     * content width for trunction logic
+     */
+    .tab {
+      flex: 0 0 auto;
+    }
+  }
+
+  .tab,
+  .tab-menu {
+    cursor: pointer;
+    font-size: var(--terra-tabs-font-size, 0.857rem);
+    line-height: var(--terra-tabs-line-hight, 1.167);
+    padding: var(--terra-tabs-padding, 0.429rem 1.071rem 0.357rem);
```

Comment:
Talked with @neilpfeiffer about this, let's update this padding to have the height of the tab compute to 40px. May need to adjust the line-height to compute to 30px; This will match the design of legacy tabs better.